course7/homeworks/waleed/ls/main.cc

```diff
@@ -0,0 +1,16 @@
+#include <boost/filesystem.hpp>
+
+#include <iostream>
+
+int main(int argc, char** argv)
+{
+	boost::filesystem::path CurrPath = boost::filesystem::current_path();
```

Comment:
Inconsistent naming. This variable is starts with CAPS, while argc/argv start with lower caps.